mcsema/BC/Function.cpp

```diff
@@ -579,6 +742,9 @@ static void AllocStackVars(llvm::BasicBlock *bb,
     // TODO(kumarak): Alignment of `alloca`s?
     s->llvm_var = ir.CreateAlloca(array_type, array_size, s->name);
   }
+
+  cfg_func->rsp_var = ir.CreateAlloca(llvm::Type::getInt64Ty(*gContext), llvm::ConstantInt::get(gWordType, 1), "rsp_store");
```

Comment:
Put the `rsp_var` and `rbp_var` in the `TranslationContext` class as opposed to the `cfg_func`. Also name them, like `stack_pointer_var` and `frame_pointer_var`, or `sp_backup_var` and `fp_backup_var`.

Reply:
The `rsp_var` and `rbp_var` are local variables and used for saving and restoring the stack/base pointers. Similar to other local variables I have put them in the `cfg_func`.